src/appengine/handlers/cron/project_setup.py

```diff
@@ -127,6 +127,9 @@ def job_name(self, project_name):
     'address', ['honggfuzz', 'engine_asan'],
     minimize_job_override=LIBFUZZER_ASAN_JOB)
 
+SYZKALLER_KASAN_JOB = JobInfo('syzkaller_kasan_', 'syzkaller', 'address',
+                              ['syzkaller', 'engine_kasan', 'prune'])
```

Comment:
Skip changing project_setup.py, this file is used for automated job setup. right now, we will manually define job and once it is stable, then move it here.

Reply:
Done.